Blackjack/Model/Card.swift

```diff
@@ -37,11 +37,15 @@ struct Card {
     }
         
     init(value: Int, suit: String, title: String? = nil) {
-        self.value = value
+        self.value = calcValue(value)
```

Comment:
Surely can just be direct assignment? It's trivial enough

```suggestion
        self.value = value > 10 ? 10 : value
```

Reply:
i did this function to also included the ace condition making it 11. tho at the time i wasnt sure how the ace = 11 or 1 thing worked so i didnt include the calculation.